minifi/minifi-bootstrap/src/test/java/org/apache/nifi/minifi/bootstrap/service/BootstrapCodecTest.java

```diff
@@ -172,13 +186,42 @@ void testCommunicateShouldHandleShutdownCommand() throws IOException {
     void testCommunicateShouldHandleReloadCommand() throws IOException {
         InputStream inputStream = new ByteArrayInputStream("RELOAD".getBytes(StandardCharsets.UTF_8));
 
-        PeriodicStatusReporterManager periodicStatusReporterManager = mock(PeriodicStatusReporterManager.class);
         ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-        BootstrapCodec bootstrapCodec = new BootstrapCodec(runner, inputStream, outputStream);
-        when(runner.getPeriodicStatusReporterManager()).thenReturn(periodicStatusReporterManager);
 
-        bootstrapCodec.communicate();
+        bootstrapCodec.communicate(inputStream, outputStream);
+
+        assertEquals(OK, outputStream.toString().trim());
+    }
+
+    @Test
+    void shouldHandleUpdateConfigurationCommand() throws IOException {
```

Comment:
While I really like the "should" prefixed test naming for the sake of consistency we should follow the convention of the already existing tests.